cli/cmd/compliance_aws.go

```diff
@@ -243,6 +245,96 @@ To show recommendation details and affected resources for a recommendation id:
 			return nil
 		},
 	}
+
+	// complianceAwsDisableReportCmd represents the disable-report sub-command inside the aws command
+	// experimental feature
+	complianceAwsDisableReportCmd = &cobra.Command{
+		Use:     "disable-report <report_type>",
+		Aliases: []string{"disable"},
+		Hidden:  true,
+		PreRunE: func(_ *cobra.Command, args []string) error {
+			switch args[0] {
+			case "CIS":
+				args[0] = fmt.Sprintf("AWS_%s_S3", args[0])
+				return nil
+			case "AWS_CIS_S3":
+				return nil
+			default:
+				return errors.New("CIS is the only supported report type")
+			}
+		},
+		Args: cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+
+			schema, err := fetchCachedAwsComplianceReportSchema(args[0])
+			if err != nil {
+				return errors.Wrap(err, "unable to aws azure compliance report schema")
```

Comment:
Should this say `get aws`?
```suggestion
				return errors.Wrap(err, "unable to get aws compliance report schema")
```

Reply:
it should, thanks